README.md

```diff
@@ -15,13 +15,13 @@ I'm a recent computer science graduate passionate about machine learning and adv
 Project source code: [https://github.com/jarviscanada/jarvis_data_eng_SiddarthKrishnan](https://github.com/jarviscanada/jarvis_data_eng_SiddarthKrishnan)
 
 
-**Cluster Monitor** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_SiddarthKrishnan/tree/master/linux_sql)]: Set up monitoring scripts for a linux cluster to track usage statistics for several nodes. These nodes are connected locally and the script will periodically populate a table with each node's current usage information. This project helped get acquainted with the linux filesystem hierarchy. It was apparent that virutal memory comprises process info, memory, CPU scheduling and other statistics. Logging memory statistics was a good exercise in understanding file paritions, which manage the storage space.
+**Cluster Monitor** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_SiddarthKrishnan/tree/master/linux_sql)]: Set up monitoring scripts for a linux cluster to track usage statistics for several nodes. These nodes are connected locally and the script our periodically populates a table with each node's current usage information. This project helped get acquainted with the linux filesystem hierarchy. It was apparent that virutal memory comprises process info, memory, CPU scheduling and other statistics. Logging memory statistics was a good exercise in understanding file paritions, which manage the storage space on the OS.
 
 **Core Java Apps** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_SiddarthKrishnan/tree/masterhttps://github.com/jarviscanada/jarvis_data_eng_SiddarthKrishnan/tree/master/core_java)]:
       
-  - Twitter App: In the process of making CRUD app for Twitter to learn APIs.
-  - JDBC App: Used JDBC to access database from the RDBMS to abstract CRUD operations. Used the data access object (DAO) pattern in code to execute SQL statements. The DAO pattern along with java.sql api were used in implementation to perfrom operations on an existing database. Our DAO classes hides how objects are persisted in the database. I also explored another the repository pattern and its advantages in this small project.
-  - Grep App: Implemented the bash grep function in a java application. This app made use of java regex classes and file handling to test its usage for text files.
+  - Twitter App: Currently working on CRUD app for Twitter to learn APIs. Using HTTP and json within java.
+  - JDBC App: The client side JDBC package java.sql was used to access RDBMS to abstract CRUD operations. The data access object (DAO) pattern was followed in our code to execute SQL statements. The DAO pattern along with java.sql api were used in implementation to perfrom operations on an existing database of customers and orders. Our DAO classes hide how objects are persisted in the database. I also explored the repository pattern and its advantages in this assignment.
```

Comment:
Developed/Designed/Implemeneted XYZ using A, B, C for purposes. features, algo, design pattern, achievements